### PR TITLE
[instruqt] helm chart upgrade

### PR DESCRIPTION
**Pull request by pabloopez**
Multiple changes to update how we deploy the agent with Helm:
* Update to new chart
* Update chart parameter to use new fields based on map [here](https://github.com/sysdiglabs/charts/blob/master/charts/sysdig-deploy/scripts/migrate_values.py)
* remove `USE_NODE_IMAGE_ANALYZER`
* Change regions logic and naming to be aligned with docs

TODO: check `agent.auditLog.auditServerUrl` and `agent.auditLog.auditServerPort` for our envs

### CODE REVIEW

**Comment by pabloopez** [comment]
This changes were not planned to be applied, but draios/instruqt#460 required to use the new chart.

File: common/prepare-track/init.sh [hunk 1]
```diff
@@ -288,17 +319,23 @@ function installation_method () {
...
 # Usage:
-#   install_agent ${CLUSTER_NAME} ${ACCESS_KEY} ${COLLECTOR}
+#   install_agent ${CLUSTER_NAME} ${ACCESS_KEY} ${COLLECTOR} ${HELM_REGION_ID}
 ##
 function install_agent () {
 
     CLUSTER_NAME=$1
     ACCESS_KEY=$2
     COLLECTOR=$3
+    HELM_REGION_ID=$4
```
Comment (pmusa) [inline common/prepare-track/init.sh:329]: I don't like this variable name, but I understand it (no need to change). It got me wondering if the same setting exists for the other installation methods. I found a `REGION` setting in [here](https://docs.sysdig.com/en/docs/installation/sysdig-agent/agent-configuration/understand-the-agent-configuration/#environment-variables-for-agent-configuration-file) and [here](https://docs.sysdig.com/en/docs/installation/sysdig-agent/agent-installation/quick-install-sysdig-agent-/#configuration-options), but I am not sure if it overwrites the `COLLECTOR` setting in docker and I don't see a region setting in the host-based one. Worth asking engineering as it would simplify the script.

**Review by pmusa — APPROVED** [review]
LGTM, left minor comments.

File: common/prepare-track/init.sh [hunk 2]
```diff
@@ -177,19 +208,19 @@ function select_region () {
             exit 0
             ;;
         1)
-            REGION="GCP US West (default)"
+            REGION="US West GCP (Dallas) - us4"
```
[the remark this answers is not in the document]
Comment (pabloopez) [reply common/prepare-track/init.sh]: hehe thanks, I detected this 10 minutes ago. it was fixed in https://github.com/draios/instruqt-assets/pull/82/commits/d020e6729dfcefe29ae9c1e7470ad636afd0ad7b

File: common/prepare-track/init.sh (hunk 1 above)
Comment (pabloopez) [reply common/prepare-track/init.sh:329]: That was my initial idea too, I would like to be the helm installer parameters to be aligned with docker and linux installer. 

But they are not, I checked both the docs and product installer instructions and there's no alignment. This is the best we can have for now.

**Comment by pabloopez** [comment]
I am not done yet with the tests.

**Comment by pabloopez** [comment]
Test results:

```

Configuring Sysdig SECURE API
  Visit https://secure.sysdig.com/#/settings/user to retrieve your Sysdig SECURE API Token.
  Insert here your Sysdig SECURE API Token: 17f43073-96e4-4221-9117-65ac17eaa84d
  Testing connection to API...   OK

Testing if Sysdig Agent is running correctly...
  FAIL
  Agent failed to connect to back-end. Check your Agent Key.

Some errors were detected configuring this lab. Please, run again this script with:
   /usr/bin/bash /root/prepare-track/init.sh

You can ask for help using Intercom or get in touch with us at team-training@sysdig.com
root@controlplane:~# ls /opt/sysdig/
MONITOR_API_ENDPOINT  SECURE_API_ENDPOINT  agent_deploy_date  helm_install.out  user_data_SECURE_API_OK
root@controlplane:~# cat /opt/sysdig/helm_install.out
"sysdig" already exists with the same configuration, skipping
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "sysdig" chart repository
Update Complete. ⎈Happy Helming!⎈
namespace/sysdig-agent created
NAME: sysdig-agent
LAST DEPLOYED: Fri Jan 20 12:09:03 2023
NAMESPACE: sysdig-agent
STATUS: deployed
REVISION: 1
NOTES:
The agent for Sysdig Secure DevOps Platform is spinning up on each node in your
cluster. After a few seconds, you should see your hosts appearing in the
Sysdig Agent Health & Status Dashboard.

Links for your deployment:
  * Sysdig Monitor: https://app.sysdigcloud.com/#/dashboard-template/view.sysdig.agents?last=10
  * Sysdig Secure: https://secure.sysdig.com/#/data-sources/agents
root@controlplane:~# kubectl get pods -n sysdig-agent
NAME                                          READY   STATUS    RESTARTS   AGE
sysdig-agent-kspmcollector-654d94f746-zchqw   1/1     Running   0          8m12s
sysdig-agent-node-analyzer-pv9ck              4/4     Running   0          8m12s
sysdig-agent-node-analyzer-tw5jv              4/4     Running   0          8m12s
sysdig-agent-xvjdw                            1/1     Running   0          8m12s
sysdig-agent-xw25j                            1/1     Running   0          8m12s
root@controlplane:~# 
```

**Comment by pabloopez** [comment]
TLDR: probably something changed in agent logs that we check to see if the installation was successful. 

The changes lgtm, agent is running with the new components. The problem is with our script checks.

**Comment by pabloopez** [comment]
Summary of the last changes after @pmusa review: this scripts were tested only with `us1` region, now it should work with all regions available today.

I tested: `us1`, `us2`, `us4`, `eu1`, `au1` and all are working fine to deploy agents and cloud accounts in aws.

Ready to merge now and be used in draios/instruqt#460.
